core/src/main/scala/org/apache/spark/util/ClosureCleaner.scala

```diff
@@ -18,13 +18,16 @@
 package org.apache.spark.util
 
 import java.io.{ByteArrayInputStream, ByteArrayOutputStream}
-import java.lang.invoke.SerializedLambda
+import java.lang.invoke.{MethodHandleInfo, SerializedLambda}
 
+import scala.collection.JavaConverters._
 import scala.collection.mutable.{Map, Set, Stack}
 import scala.language.existentials
```

Comment:
cc @kiszk note that the Spark 2.4 version of this file imports `scala.language.existentials`